docs/reference/sem-command-line-tool.md

```diff
@@ -173,7 +173,23 @@ must be connected to the organization to which it belongs.
 Aditionally a `secret` can be associated with a single project. This is 
 called a project-level secret. In order to manage project-level secrets
 you must provide the project name or id of the associated project with 
-`-i` or `-p` flags, or use a different yaml `kind` described [here](reference/secrets-yaml-reference).
+`-i` or `-p` flags, or use a different yaml `kind` described [here](/reference/secrets-yaml-reference).
+
+#### Deployment Targets
+
+The `deployment-targets` serve as stringent control mechanisms, determining who
+can activate individual pipelines and specify the conditions required for
+their initiation. These mechanisms offer elevated control, allowing you to
+manage the execution of particular promoted pipelines or choose git references,
+including branches and tags.
+
+Each `deployment-target` is initially created within a single project. However,
+it can later be linked with numerous [promotions](/essentials/deploying-with-promotions).
```

Comment:
Let's change this into:

Each `deployment-target` is created within a single project and it can be linked with numerous [promotions](/essentials/deploying-with-promotions) within that project.